### ISSUE TITLE
Firefox support

### ISSUE THREAD

**Issue by BGM99** [JasonZigelbaum/jqbx-issues#219]
joining a room is impossible with Firefox

Error:
Prevented navigation to “jqbx://room/...” due to an unknown protocol.

**Comment by JasonZigelbaum** [comment]
Can you try visiting the website instead:

https://app.jqbx.fm

Does that work?

**Comment by BGM99** [comment]
I used that URL.
Is this really just a local problem on my machine?

**Comment by JasonZigelbaum** [comment]
Sorry I'm not sure I understand. Here's my understanding:

1. The jqbx://room url is only meant to be used if you have the JQBX
desktop app installed
2. The website: https://app.jqbx.fm doesn't use those urls, it's just a
website that you put into your web browser.

Are you telling me that when you go to: https://app.jqbx.fm you get
redirected to a jqbx://room link?

**Comment by BGM99** [comment]
No, im not getting redirected. The Error I posted in the beginning was in the Console. I don't think this is the actual Problem.
The Problem is that I cannot use the App in my Firefox Nightly Browser (only in my Chrome).
After I Login it gets always Stuck in the "LOGGING IN ..." View.
